src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/UidManager.cs

```diff
@@ -74,7 +74,7 @@ public UidManager() : base(SR.SharedResourceManager)
         /// </summary>
         public override bool Execute()
         {
-            TaskHelper.DisplayLogo(Log, SR.Get(SRID.UidManagerTask));
+            TaskHelper.DisplayLogo(Log, "UidManagerTask");
```

Comment:
The string value here was just "UidManager".

Reply:
I don't understand this resolution.

Reply:
Unless you were pre-resolving, this isn't correct still.

Reply:
My bad, I was in some sort of half refreshed state, this is good now.